src/background/index.js

```diff
@@ -1,56 +1,63 @@
-import { LOOKUP_PLACE, SCAN_PARAGRAPH } from '../extensionMessageTypes';
+import {
+    CONTEXT_MENU_TITLE,
+    LOOKUP_PLACE_TITLE,
+    SCAN_PARAGRAPH_TITLE,
+    CONTEXT_MENU_ID,
+    LOOKUP_PLACE_ID,
+    SCAN_PARAGRAPH_ID
+} from './constants.js';
+import {
+    LOOKUP_PLACE,
+    SCAN_PARAGRAPH
+} from '../extensionMessageTypes';
 
 function lookupPlaceListener({ selectionText }) {
-    chrome.tabs.query(
-        {
-            active: true,
-            currentWindow: true,
-        },
-        (tabs) => {
-            chrome.tabs.sendMessage(tabs[0].id, {
-                message: LOOKUP_PLACE,
-                data: selectionText,
-            }); // Handle The Response
-        },
-    );
+    chrome.tabs.query({
+        active: true,
+        currentWindow: true,
+    },
+    ([firstTab]) => {
+        chrome.tabs.sendMessage(firstTab.id, {
+            message: LOOKUP_PLACE,
+            data: selectionText,
+        }); // Handle The Response
+    });
 }
 
 function scanParagraphListener({ selectionText }) {
-    chrome.tabs.query(
-        {
-            active: true,
-            currentWindow: true,
-        },
-        (tabs) => {
-            chrome.tabs.sendMessage(tabs[0].id, {
-                message: SCAN_PARAGRAPH,
-                data: selectionText,
-            }); // Handle The Response
-        },
-    );
+    chrome.tabs.query({
+        active: true,
+        currentWindow: true,
+    },
+    ([firstTab]) => {
+        chrome.tabs.sendMessage(firstTab.id, {
+            message: SCAN_PARAGRAPH,
+            data: selectionText,
+        }); // Handle The Response
+    });
 }
 
 const listenerFunctionMap = {
-    lookupPlaceWanderland: lookupPlaceListener,
-    scanParagraphWanderland: scanParagraphListener,
+    [LOOKUP_PLACE_ID]: lookupPlaceListener,
```

Comment:
hmm if id's like these aren't going to leave the page, I'd still make them constants but leave the constants defined at the top of this file (below imports)